examples/getting_started/quickstart.ipynb

```diff
@@ -0,0 +1,535 @@
+      "  \"population\": 13929286,\n",
+      "  \"area_km2\": 2191,\n",
+      "  \"landmarks\": [\n",
+      "    {\n",
+      "      \"name\": \"Tokyo Tower\",\n",
+      "      \"type\": \"Tower\",\n",
+      "      \"height_m\": 333\n",
+      "    },\n",
+      "    {\n",
+      "      \"name\": \"Senso-ji Temple\",\n",
+      "      \"type\": \"Temple\",\n",
+      "      \"location\": \"Asakusa\"\n",
+      "    },\n",
+      "    {\n",
+      "      \"name\": \"Shibuya Crossing\",\n",
+      "      \"type\": \"Intersection\",\n",
+      "      \"description\": \"Famous pedestrian scramble\"\n",
+      "    }\n",
+      "  ],\n",
+      "  \"transportation\": {\n",
+      "    \"subway_system\": true,\n",
+      "    \"major_airports\": [\n",
+      "      {\n",
+      "        \"name\": \"Narita International Airport\",\n",
+      "        \"IATA_code\": \"NRT\"\n",
+      "      },\n",
+      "      {\n",
+      "        \"name\": \"Haneda Airport\",\n",
+      "        \"IATA_code\": \"HND\"\n",
+      "      }\n",
+      "    ]\n",
+      "  },\n",
+      "  \"culture\": {\n",
+      "    \"cuisine\": [\n",
+      "      \"Sushi\",\n",
+      "      \"Ramen\",\n",
+      "      \"Tempura\"\n",
+      "    ],\n",
+      "    \"traditional_events\": [\n",
+      "      \"Cherry Blossom Festival\",\n",
+      "      \"Gion Matsuri\"\n",
+      "    ]\n",
+      "  }\n",
+      "}\n"
+     ]
+    }
+   ],
+   "source": [
+    "@openai.call(\"gpt-4o-mini\", json_mode=True)\n",
+    "@prompt_template(\"Provide information about {city} in JSON format\")\n",
+    "def city_info(city: str): ...\n",
+    "\n",
+    "\n",
+    "response = city_info(\"Tokyo\")\n",
+    "print(response.content)  # This will be a JSON-formatted string"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Note that not all providers have an explicit JSON mode. For those providers, we attempt to instruct the model to provide JSON; however, there is no guarantee that it will output only JSON (it may start with some text like \"Here is the JSON: ...\"). This is where Output Parsers can be useful.\n",
+    "\n",
+    "For more information on JSON mode, refer to our [Learn documentation on JSON Mode](https://docs.mirascope.io/learn/json-mode)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Response Models\n",
+    "\n",
+    "Response models allow you to structure and validate the output from LLMs."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {
+    "ExecuteTime": {
+     "end_time": "2024-09-06T14:00:25.412924Z",
+     "start_time": "2024-09-06T14:00:24.661722Z"
+    }
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The capital of Japan is Tokyo\n"
+     ]
+    }
+   ],
+   "source": [
+    "from pydantic import BaseModel\n",
+    "\n",
+    "\n",
+    "class Capital(BaseModel):\n",
```

Comment:
for this one, rather than showing how to generate a response model, we should show how to extract unstructured information into a structured format. For example:

```python
from pydantic import BaseModel

class Capital(BaseModel):
    city: str
    country: str

@openai.call("gpt-4o-mini", response_model=Capital)
@prompt_template("{query}")
def extract_capital(query: str): ...

capital = extract_capital("The capital of France is Paris")
print(capital)
```